README.md

```diff
@@ -3931,7 +3931,7 @@ Other Style Guides
 
 **Podcasts**
 
-  - [JavaScript Air](https://javascriptair.com/)
+  - [JavaScript Air](http://audio.javascriptair.com/)
```

Comment:
is this not available using https?

Reply:
@Yash-Ambekar you marked this as resolved, but it's not resolved. can you answer the question?